lib/html.rb

```diff
@@ -73,6 +74,7 @@ def self.scope_help(scope)
   end
 
   def self.diff_to_object(diff)
+    diff.encode!('UTF-8', invalid: :replace, undef: :replace)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.